executor/runtime/docker/docker_isolate.go

```diff
@@ -82,6 +87,11 @@ func workloadIsolated(ctx context.Context, taskID, host string) bool {
 		return true
 	}
 
+	if resp.StatusCode == 404 {
+		// Only log unexpected status codes below
```

Comment:
Is this expected? why would we get a 404 after all these attempts?

Reply:
404 is isolate's response when it hasn't completed the isolation yet.